src/components/modals/QrCodeModal.tsx

```diff
@@ -59,7 +59,13 @@ export function QrCodeModal(props: Props) {
           // New core versions have the callbacks on the request:
           pending.watch('state', state => {
             if (state === 'started') {
-              setUsername(pending.username ?? lstrings.missing_username)
+              setUsername(
+                pending.username ??
+                  sprintf(
+                    lstrings.guest_account_id_1s,
+                    pending.id.slice(pending.id.length - 3)
```

Comment:
The `id` here is `lobbyId`, not the `loginId`. We don't actually know what the `loginId` will be at this point in the process, so there is nothing useful we can put in the "(...)" part. You might as well put the string "???" as the loginId, since at least that would be more honest than putting in a random unrelated string.

Another option is to just set the username to "" once login starts. Then when the modal says "Logging into account %s", it will just render as "Logging into account", which makes sense.